docs/csharp/misc/cs0575.md

```diff
@@ -10,9 +10,9 @@ ms.assetid: e8f20960-94a6-41d0-807c-d49ad198ccf6
 ---
 # Compiler Error CS0575
 
-Only class types can contain destructors  
+Only class types can contain finalizers  
```

Comment:
This line is the compiler-produced text, so shouldn't be changed.

Reply:
Fixed.